search/search_data_blending_with_gemini_summarization.ipynb

```diff
@@ -332,26 +332,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 84,
-   "id": "0093a8b5",
+   "execution_count": null,
+   "id": "52e6ce64",
    "metadata": {},
    "outputs": [],
    "source": [
+    "from google.protobuf.json_format import MessageToDict\n",
+    "\n",
+    "# protobuf object to dict\n",
+    "response_json = MessageToDict(response._pb)\n",
     "retrieved_data: List[str] = []\n",
     "\n",
-    "for result in response.results:\n",
-    "    data = result.document.derived_struct_data\n",
+    "for result in response_json[\"results\"]:\n",
+    "    data = result[\"document\"][\"derivedStructData\"]\n",
     "    if not data:\n",
     "        continue\n",
     "\n",
     "    snippets: List[str] = [\n",
-    "        re.sub(\"<[^>]*>\", \"\", snippet_item.get(\"snippet\", \"\"))\n",
-    "        for snippet_item in data.get(\"snippets\", [])\n",
-    "        if snippet_item.get(\"snippet\")\n",
+    "        re.sub(\"<[^>]*>\", \"\", snippet_item.get(\"content\", \"\"))\n",
+    "        for snippet_item in data.get(\"extractive_answers\", [])\n",
+    "        if snippet_item.get(\"content\")\n",
```

Comment:
This Extractive answer type of content is different than `snippets`

https://cloud.google.com/generative-ai-app-builder/docs/snippets#examples

Can you change this code to make `extractive_answers` as a backup if the `snippets` field is empty? Not every type of data can use `extractive_answers`